src/main/java/io/confluent/examples/streams/microservices/util/MicroserviceUtils.java

```diff
@@ -49,17 +51,35 @@ public static String parseArgsAndConfigure(String[] args) {
     return bootstrapServers;
   }
 
-  public static Properties baseStreamsConfig(String bootstrapServers, String stateDir,
-      String appId) {
+  public static Properties baseStreamsConfig(final String bootstrapServers,
+                                             final String stateDir,
+                                             final String appId) {
+    final boolean enableEOS = false;
```

Comment:
Nit: This local variable can be removed. Pass in `false` directly in next line.

Reply:
Done